style.css

```diff
@@ -1,4 +1,12 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat|Oswald');
+
+* {
```

Comment:
web-kit extensions are not too compatible with other browsers that do not use blink,
https://developer.mozilla.org/en-US/docs/Web/CSS/Webkit_Extensions

use margin-top, margin-right, margin-left, margin-bottom instead, or something along those lines

Reply:
I added it cause my browser gave it default values. I guess I can leave it out for now and we can look into it later once we have the content done.